### PR TITLE
Mention the meta-programming guide

### LINKED ISSUE #447: Link to "Meta Programming in Elixir"
I have checked the links, and this guide is getting very little attention, 
the only place where it is actually being linked to is this post:
http://elixir-lang.org/blog/2014/04/21/elixir-v0-13-0-released/

and that's it.
I think we should at least include it at the end of the Getting Started guide
http://elixir-lang.org/getting-started/where-to-go-next.html#21.1-build-your-first-elixir-project
Aftet the Mix & OTP guide is mentioned.

### PR DESCRIPTION
**Pull request by whatyouhide**
Mentioned the meta-programming guide in the last chapter of the Getting Started guide, "Where to go next". This should close #447.

### CODE REVIEW

**Comment by josevalim** [comment]
:heart: :green_heart: :blue_heart: :yellow_heart: :purple_heart:

File: getting-started/where-to-go-next.markdown [hunk 1]
```diff
@@ -20,6 +20,12 @@ We have written a guide that covers how to build an Elixir application, with its
 
 * [Mix and OTP](/getting-started/mix-otp/introduction-to-mix.html)
 
+## Meta-programming
+
+Elixir is an extensible and very customizable programming language thanks to its meta-programming support. Most meta-programming in Elixir is done through macros, which are very useful in several situations, especially for writing DSLs. We have written a short guide that explains the basic mechanisms behind macros and shows how to write macros and use them to create DSLs:
+
+* [Meta-programming in Elixir](/getting-started/meta/quote-and-unquote.html)
+
```
Comment (eksperimental) [inline getting-started/where-to-go-next.markdown]: I think it's better to link to: [Meta-programming in Elixir](/getting-started/meta/)

for reasons discussed here https://github.com/elixir-lang/elixir-lang.github.com/pull/446#issuecomment-73217804

which will end up being linked to the index, when we have time to create it.

**Comment by eksperimental** [comment]
sweet as!

File: getting-started/where-to-go-next.markdown (hunk 1 above)
Comment (josevalim) [inline getting-started/where-to-go-next.markdown]: Nah, it is fine as it is right now. The index is also on the sidebar if people are interested.

**Pull request merged**